src/app/screens/Publishers/index.tsx

```diff
@@ -90,21 +103,107 @@ function Publishers() {
+            />
+          ) : (
+            <p className="dark:text-white"> {t("used.no_info")}</p>
+          )}
+        </>
+      )}
+      {publishers.length === 0 && hasTips && (
+        <>
+          <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
+            {t("tips.title")}
+          </h2>
+          <p className="mb-6 text-gray-500 dark:text-neutral-500">
+            {t("tips.description")}
+          </p>
+          <div className="grid gap-4 sm:grid-cols-1 lg:grid-cols-2">
+            {hasTip(TIPS.TOP_UP_WALLET) && (
+              <CloseableCard
+                handleClose={() =>
+                  updateSetting({
+                    tips: tips.filter((tip) => tip !== TIPS.TOP_UP_WALLET),
+                  })
+                }
+                title={t("tips.top_up_wallet.title")}
+                description={t("tips.top_up_wallet.description")}
+                buttons={[
+                  <Button
+                    key={1}
+                    label={t("tips.top_up_wallet.label1")}
+                    primary
+                    onClick={() => {
+                      navigate("/receive");
+                    }}
+                  />,
+                  <a key={2} href="https://getalby.com/topup">
+                    <Button label={t("tips.top_up_wallet.label2")} />
+                  </a>,
+                ]}
+              />
+            )}
+            {hasTip(TIPS.PIN) && (
+              <CloseableCard
+                handleClose={() =>
+                  updateSetting({
+                    tips: tips.filter((tip) => tip !== TIPS.PIN),
+                  })
+                }
+                title={t("tips.pin.title")}
+                description={[
+                  t("tips.pin.description1"),
+                  t("tips.pin.description2"),
+                  t("tips.pin.description3"),
+                ]}
+              />
+            )}
+            {hasTip(TIPS.DEMO) && (
+              <CloseableCard
+                handleClose={() =>
+                  updateSetting({
+                    tips: tips.filter((tip) => tip !== TIPS.DEMO),
+                  })
+                }
+                title={t("tips.demo.title")}
+                description={t("tips.demo.description")}
+                buttons={[
+                  <a key={1} href="https://getalby.com/demo">
+                    <Button label={t("tips.demo.label1")} primary />
+                  </a>,
+                ]}
+              />
+            )}
+            {hasTip(TIPS.ADDRESS) && (
+              <CloseableCard
+                handleClose={() =>
+                  updateSetting({
+                    tips: tips.filter((tip) => tip !== TIPS.ADDRESS),
+                  })
+                }
+                title={t("tips.address.title")}
+                description={t("tips.address.description")}
+                buttons={[
+                  <a key={1} href="https://getalby.com/demo">
```

Comment:
Is this link right? the address tip goes to the demo

Reply:
The whole ln-address handling is unclear currently. This box is currently not activated.

Reply:
i can remove the address stuff from the PR if desired.  Just put it here as a placeholder for when its ready.